test/e2e/integreatly_test.go

```diff
@@ -488,6 +488,7 @@ func integreatlyManagedTest(t *testing.T, f *framework.Framework, ctx *framework
 	productOperators := map[string]string{
 		"3scale":               string(integreatlyv1alpha1.OperatorVersion3Scale),
 		"amqonline":            string(integreatlyv1alpha1.OperatorVersionAMQOnline),
+		"apicurito":            string(integreatlyv1alpha1.VersionApicurito),
```

Comment:
ah, the problem with the e2e is here, this should be `"apicurito": string(integreatlyv1alpha1.OperatorVersionApicurito),`